src/components/common/TmSessionHardware.vue

```diff
@@ -13,8 +13,16 @@
         </a>
       </div>
       <div class="tm-session-main">
-        <HardwareState :loading="status === `detect` ? true : false">
-          Please plug in your Ledger&nbsp;Nano&nbsp;S and open the Cosmos app
+        <HardwareState :loading="status === `connect` ? false : true">
+          <template v-if="status === `connect` || status === `detect`">
+            Please plug in your Ledger&nbsp;Nano&nbsp;S and open the Cosmos app
+          </template>
+          <template v-if="status === `confirmAddress`">
+            Signin in with the address
+            <span class="address">{{ address }}</span
+            >.<br />
+            Please confirm on your Ledger.
```

Comment:
is there an action associated with this confirm message? if no, it should say "Please confirm that your address is correct" or something like that.

Reply:
Yes it is associated with an action on the Ledger. Please test these critical features on your Ledger if they come up.